### DISCUSSION TITLE
Shortcut key to collapse/expand when project groupings

### DISCUSSION THREAD

**Discussion by mbbroberg** [community/community#108072] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

I'm a huge user of Projects and it's increasingly difficult to navigate around my `strategy` board. It includes 10 different group sections with 100+ items in them, and I went down the path of looking at hotkeys to expand/collapse all groups. I didn't see any, but maybe there's a query string option (?group=collapse) I'm not familiar with. 

Adding something like this would be quite helpful -- thanks for considerations.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by JonathanPalmerGD** [comment]
I would absolutely love this. 
Having every category visible all the time does not meet many ways a project's organization can be structured!

Good ask!

**Comment by chriseaton** [comment]
This and/or buttons/menu options. Need a "collapse/expand all" option.
